docs/proposals/2170-kubeflow-training-v2/README.md

```diff
@@ -279,6 +279,43 @@ type TrainJob struct {
 	Status TrainJobStatus `json:"status,omitempty"`
 }
 
+const (
+	// TrainJobSuspended means the TrainJob is suspended.
+	TrainJobSuspended string = "Suspended"
+
+	// TrainJobCompleted means that the actual jobs have completed its execution.
+	TrainJobCompleted string = "Completed"
+
+	// TrainJobFailed means that the actual jobs have failed its execution.
+	TrainJobFailed string = "Failed"
+
+	// TrainJobCreated means that the actual jobs creation has succeeded.
+	TrainJobCreated string = "Created"
+)
+
+const (
+	// TrainJobSuspendedReason is the "Suspended" condition reason.
+	// When the TrainJob is suspended, this is added.
+	TrainJobSuspendedReason string = "Suspended"
+
+	// TrainJobResumedReason is the "Suspended" condition reason.
+	// When the TrainJob suspension is changed from True to False, this is added.
+	TrainJobResumedReason string = "Resumed"
```

Comment:
@tenzen-y @kannon92 Do we have that status reason in JobSet or Job ?

Reply:
Yes, we have. In the batch/v1 Job level, "JobResumed" is used as a reason for the "Suspended" condition type.
In the JobSet level, "ResumeJobs" is used as a reason for the "Suspended" condition type.